### DISCUSSION TITLE
Cut at Extract Keyframe

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2017] [Q&A]
Whenever I start at a keyframe, my video always starts at the keyframe before that. How do I fix that?

**Comment by mifi** [comment]
> Each segment's start cut time normally (but not always) will be "rounded" to the nearest previous keyframe. This means that you often have to move the start cut time to few frames after the desired keyframe.

https://github.com/mifi/lossless-cut/blob/master/issues.md#faq
